### PR TITLE
containers: replace sed calls with dedicated function

### PR DESCRIPTION
**Pull request by asmorodskyi**
VR : http://autobot.qa.suse.de/tests/2347

### CODE REVIEW

File: lib/containers/utils.pm [hunk 1]
```diff
@@ -161,7 +161,7 @@ sub container_set_up {
     assert_script_run "mkdir -p $dir/BuildTest/templates";
     assert_script_run "curl -f -v " . data_url('containers/app.py') . " > $dir/BuildTest/app.py";
     assert_script_run "curl -f -v " . data_url("containers/$file") . " > $dir/BuildTest/Dockerfile";
-    assert_script_run "sed -i 's,baseimage_var,$base,1' $dir/BuildTest/Dockerfile" if defined $base;
+    file_content_replace("$dir/BuildTest/Dockerfile", baseimage_var => $base) if defined $base;
```
Comment (jlausuch) [inline lib/containers/utils.pm:164]: I really like this function :)

**Review by pdostal — APPROVED** [review]
Great job Anton 🚀

**Review by b10n1k — APPROVED** [review]
+1

**Comment by asmorodskyi** [comment]
decided to also do some o3 run - http://autobot.qa.suse.de/tests/2354 . waiting to finish and than will merge

**Pull request merged**